From b915077aa9c7a08a89ffe886bedd1e60ddc92a01 Mon Sep 17 00:00:00 2001
From: David Gibson <dgibson@redhat.com>
Date: Fri, 3 Jul 2015 06:45:06 +0200
Subject: [PATCH 022/100] RHEL: Disable remaining unsupported devices for ppc

Message-id: <1435905906-24552-4-git-send-email-dgibson@redhat.com>
Patchwork-id: 66688
O-Subject: [RHEL-7.2 qemu-kvm-rhev PATCH 3/3] RHEL: Disable remaining unsupported devices for ppc
Bugzilla: 1191845
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Laurent Vivier <lvivier@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>

This is a downstream only patch for the configuration, removing the
remaining qemu devices which we don't want to support on Power.

CONFIG_ISA_BUS: No ISA on Power
CONFIG_ISA_MMIO: This option doesn't actually do anything, anyway
CONFIG_I8259: With no ISA, we don't need the legacy i8259 PIC
CONFIG_PLATFORM_BUS: Only used on ppc embedded machine types
CONFIG_SERIAL_ISA: No ISA, so no legacy serial ports
CONFIG_ISA_TESTDEV: No ISA

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1191845

Signed-off-by: David Gibson <dgibson@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/ppc64-softmmu.mak | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/default-configs/ppc64-softmmu.mak b/default-configs/ppc64-softmmu.mak
index e18d808..037064b 100644
--- a/default-configs/ppc64-softmmu.mak
+++ b/default-configs/ppc64-softmmu.mak
@@ -11,19 +11,12 @@ CONFIG_PCI_TESTDEV=y
 
 include sound.mak
 include usb.mak
-CONFIG_ISA_BUS=y
-CONFIG_ISA_MMIO=y
 CONFIG_VGA=y
 CONFIG_VGA_PCI=y
 CONFIG_SERIAL=y
-CONFIG_I8259=y
 CONFIG_PSERIES=y
-CONFIG_PLATFORM_BUS=y
 CONFIG_LIBDECNUMBER=y
 CONFIG_USB_OHCI=y
 # For pSeries
 CONFIG_XICS=$(CONFIG_PSERIES)
 CONFIG_XICS_KVM=$(and $(CONFIG_PSERIES),$(CONFIG_KVM))
-CONFIG_I8259=y
-CONFIG_SERIAL_ISA=y
-CONFIG_ISA_TESTDEV=y
-- 
1.8.3.1

